### PR TITLE
circular dependent fix

### PR DESCRIPTION
**Pull request by trtshen**
Minor code changes
- NotificationService.popUp's second parameter can be empty and set false as default
- reviewRating popup is better place at assessment.service, because only assessment.component will use it. (it is more reuseable if we keep them serve the generic purpose)

### CODE REVIEW

File: src/app/review-rating/review-rating.service.ts [hunk 1]
```diff
@@ -22,6 +23,7 @@ export class ReviewRatingService {
 
   constructor(
   	private request : RequestService,
+    private notification : NotificationService,
```
Comment (shawnm0705) [inline src/app/review-rating/review-rating.service.ts]: We didn't use it here

File: src/app/assessment/assessment.component.ts [hunk 2]
```diff
@@ -72,7 +73,8 @@ export class AssessmentComponent implements OnInit {
     private assessmentService: AssessmentService,
     private utils: UtilsService,
     private notificationService: NotificationService,
-    private storage: BrowserStorageService    
+    private storage: BrowserStorageService,
+    private reviewRatingService: ReviewRatingService,
```
Comment (shawnm0705) [inline src/app/assessment/assessment.component.ts]: We didn't use it here

File: src/app/shared/notification/notification.service.ts [hunk 3]
```diff
@@ -15,18 +14,33 @@ export class NotificationService {
...
-      }
-    });
+  async popUp(type, data, redirect:any = false) {
+    const component = PopUpComponent;
+    const componentProps = {
+      type,
+      data,
+      redirect,
+    };
+    const modal = await this.modalController.create(this.modalConfig({ component, componentProps }));
```
Comment (shawnm0705) [inline src/app/shared/notification/notification.service.ts]: We can use `this.modal()` here since it is defined below

**Comment by trtshen** [comment]
thanks @shawnm0705 for reviewing!

**Pull request merged**